server-worker/src/main/scala/com/aliyun/emr/rss/service/deploy/worker/Worker.scala

```diff
@@ -370,9 +370,11 @@ private[deploy] class Worker(
   ShutdownHookManager.get().addShutdownHook(
     new Thread(new Runnable {
       override def run(): Unit = {
+        workerSource.startTimer(WorkerSource.WorkerShutdownTotalTime, "WorkerShutdownTotalTime")
```

Comment:
I think these metrics will be lost from time to time since Prometheus scrapes data in a fixed period. 
The worker closed before Prometheus pulled data.

Reply:
Seems so..